app/models/concerns/uid_validation_on_update.rb

```diff
@@ -0,0 +1,7 @@
+module UidValidationOnUpdate
+  def uid_not_changed
+    if persisted? && changed.include?("uid")
```

Comment:
This is a gremlin, when I tested `uid_changed?` during implementation it didn't work. I'll change it now 👍